skynet.yaml

```diff
@@ -0,0 +1,13 @@
+name: w_module-unit-tests
+description: Unit tests
+contact: 'Frontend Frameworks Architecture / #support-frontend-architecture'
+image: drydock.workiva.net/workiva/dart_unit_test_image:1
+size: large
+timeout: 600
+
+artifacts: /testing/w_module/test-reports
+test-reports: /testing/w_module/test-reports
+
+scripts:
+  - cd /testing/w_module/
+  - ./run_tests.sh
```

Comment:
Since we already run tests in github workflows, perhaps [just no-op here](https://github.com/Workiva/w_dashboard_frugal/pull/35/files).

Reply:
What's the benefit of having them run via workflows rather than skynet?

Reply:
I believe it's so that open source contributors can see the results of a CI run (skynet requires privileged access).

Reply:
Ahhhh